syncstorage/tests/functional/test_storage.py

```diff
@@ -1485,7 +1484,6 @@ def test_batch_ttl_update(self):
         self.assertEquals(len(res), 1)
         self.assertEquals(res[0]['payload'], 'see')
 
-    @unittest2.skip("for the moment")
     def test_batch_ttl_is_based_on_commit_timestamp(self):
```

Comment:
Aye, some sort of sporadic failure on mysql backend;  swear it was working on my machine :-)